streaming/src/main/scala/io/gearpump/streaming/task/TaskActor.scala

```diff
@@ -151,7 +151,7 @@ class TaskActor(
     var done = false
 
     var count = 0
-    val start = System.currentTimeMillis()
+    val start = System.nanoTime()
```

Comment:
added comments. The nano time is local and not exposed. I'd like to keep it simple.